testing/dsp_input_sim/dsp_input_sim.cpp

```diff
@@ -145,6 +145,34 @@ void signals(zmq::context_t &context)
 
```

Comment:
calculation of num_samples has a magic number

Reply:
I just needed an arbitrary amount of samples for testing. Should I just make a comment here? Doesnt have to be a specific number.